src/server/pfs/s3/master_test.go

```diff
@@ -320,7 +320,8 @@ func masterListObjectsPaginated(t *testing.T, pachClient *client.APIClient, mini
 	// `LastModified` date is correct. A few minutes are subtracted/added to
 	// each to tolerate the node time not being the same as the host time.
 	startTime := time.Now().Add(time.Duration(-5) * time.Minute)
-	repo := tu.UniqueString("testlistobjectspaginated")
+	// bucket name cannot exceed 63 characters long
+	repo := tu.UniqueString("testLOP")
```

Comment:
This change was necessary because s3 puts a hard limit of 63 chars on the length of a bucket and the repo name was too long.